stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -78,11 +78,12 @@ private class TestCluster {
     // be started when we set TFE_ContextSetServerDef.
     for i in 1..<taskCount {
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer! =
+          TFE_GetServerDef(serverDefText, status)
```

Comment:
```suggestion
        TFE_GetServerDef(serverDefText, status)
```
Though you may be able to move it to the previous line if it's short enough.

Reply:
Changed the intendation. I was not able to move it to the previous line. It goes out of 80 chars.